components/tidb_query_expr/src/impl_string.rs

```diff
@@ -702,15 +702,15 @@ pub fn instr_utf8(s: BytesRef, substr: BytesRef) -> Result<Option<Int>> {
 
 #[rpn_fn]
 #[inline]
-pub fn find_in_set(s: BytesRef, str_list: BytesRef) -> Result<Option<Int>> {
+pub fn find_in_set<C: Collator>(s: BytesRef, str_list: BytesRef) -> Result<Option<Int>> {
     if str_list.is_empty() {
         return Ok(Some(0));
     }
 
-    let s = String::from_utf8_lossy(s);
+    use std::cmp::Ordering::*;
     let result = String::from_utf8_lossy(str_list)
```

Comment:
Please use https://docs.rs/bstr/0.2.14/bstr/trait.ByteSlice.html#method.split_str

Reply:
So should we use `bstr::ByteSlice` instead of `String::from_utf8_lossy`?

Reply:
So should we convert the `str_list` to bstr:ByteSilce instead of using `String::from_utf8_lossy`?

Reply:
It seems that `bstr::ByteSlice` can't be constructed by a `BytesRef`.